src/test/programs/federal/snap/unit.test.js

```diff
@@ -0,0 +1,349 @@
+
+    describe('a head of household', () => {
+
+      it('that is not disabled or under 18 should return false', () => {
+        expect(SNAPhelpers.hasDependentsOver12( current )).toBe(false);
+      });
+      it('that is under 13 should return false', () => {
+        current.household[0].m_age = 12;
+        expect(SNAPhelpers.hasDependentsOver12( current )).toBe(false);
+      });
+      it('that is under 18 should return false', () => {
+        current.household[0].m_age = 13;
+        expect(SNAPhelpers.hasDependentsOver12( current )).toBe(false);
+      });
+      it('that is disabled should return false', () => {
+        current.household[0].m_disabled = true;
+        expect(SNAPhelpers.hasDependentsOver12( current )).toBe(false);
+      });
+    });
+
+    describe('a spouse', () => {
+
+      beforeEach(() => { current.household[1] = { m_age: 30, m_role: 'spouse', m_disabled: false }; });
+
+      it('that is not disabled or under 18 should return false', () => {
+        expect(SNAPhelpers.hasDependentsOver12( current )).toBe(false);
+      });
+      it('that is under 13 should return false', () => {
+        current.household[1].m_age = 12;
+        expect(SNAPhelpers.hasDependentsOver12( current )).toBe(false);
+      });
+      it('that is under 18 should return false', () => {
+        current.household[1].m_age = 13;
+        expect(SNAPhelpers.hasDependentsOver12( current )).toBe(false);
+      });
+      it('that is disabled should return false', () => {
+        current.household[1].m_disabled = true;
+        expect(SNAPhelpers.hasDependentsOver12( current )).toBe(false);
+      });
+    });
+
+    describe('a regular member', () => {
+
+      beforeEach(() => { current.household[1] = { m_age: 30, m_role: 'member', m_disabled: false }; });
+
+      it('that is not disabled or under 18 should return false', () => {
+        expect(SNAPhelpers.hasDependentsOver12( current )).toBe(false);
+      });
+      it('that is under 13 should return false', () => {
+        current.household[1].m_age = 12;
+        expect(SNAPhelpers.hasDependentsOver12( current )).toBe(false);
+      });
+      it('that is under 18 should return true', () => {
+        current.household[1].m_age = 13;
+        expect(SNAPhelpers.hasDependentsOver12( current )).toBe(true);
+      });
+      it('that is disabled should return true', () => {
+        current.household[1].m_disabled = true;
+        expect(SNAPhelpers.hasDependentsOver12( current )).toBe(true);
+      });
+    });
+  });
+
+
+  // `SNAPhelpers.getChildSupportPaid()`
+  describe('`.getChildSupportPaid( timeClient )` given a time-constrained client paying', () => {
+    let current;
+    beforeEach(() => { current = cloneDeep( defaultCurrent ); });
+
+    describe('no ', () => {
+      it('that is not disabled and under 60 should return false', () => {
+        expect(SNAPhelpers.hasDisabledOrElderlyMember( current )).toBe(false);
```

Comment:
We're testing `.getChildSupportPaid()` here.

Reply:
Don't know why this is even there - the function is gone. Thanks for the catch!